apps/sasquatch/build.gradle

```diff
@@ -75,21 +75,17 @@ dependencies {
     //noinspection GradleDependency
     jcenterDependencyImplementation "com.microsoft.appcenter:appcenter-push:${appCenterSdkVersion}"
 
-    firebaseImplementation "com.google.firebase:firebase-core:${rootProject.ext.firebaseLibVersion}"
-    firebaseImplementation "com.google.firebase:firebase-messaging:${rootProject.ext.firebaseLibVersion}"
+    firebaseImplementation "com.google.firebase:firebase-core:16.0.1"
+    firebaseImplementation "com.google.firebase:firebase-messaging:17.0.0"
 
     /* Force usage this version of support annotations to avoid conflict. */
     androidTestImplementation "com.android.support:support-annotations:${rootProject.ext.supportLibVersion}"
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
     implementation 'com.android.support.test.espresso:espresso-idling-resource:3.0.1'
 }
 
-/*
- * This will also bypass google-services.json if executing assembleDebug wildcard task.
- * This is needed to build from App Center build as we don't commit google-services.json and
- * App Center build cannot detect our gradle tasks so we make it execute assembleDebug with a
- * work around. We are interested in only a vanilla build variant (no Firebase) for such builds.
- */
-if (getGradle().getStartParameter().getTaskRequests().toString().contains("Firebase")) {
+// We are interested in excluding the plugin only for the vanilla sasquatch build variant.
```

Comment:
I don't think that it's still valid.
- `This will also bypass google-services.json if executing assembleDebug wildcard task.` - it isn't true;
- `This is needed to build from App Center build as we don't commit google-services.json and App Center build cannot detect our gradle tasks so we make it execute assembleDebug with a work around.` - not true too;